### PR TITLE
SCMOD-7930: Release notes updated

### PR DESCRIPTION
**Pull request by thilagasanthosh**
https://portal.digitalsafe.net/browse/SCMOD-7930

### CODE REVIEW

File: release-notes-1.6.0.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
```
Comment (michael-bryson) [inline release-notes-1.6.0.md:6]: Add - None here

File: release-notes-1.6.0.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
 
 #### Bug Fixes
 
-- [SCMOD-5552](https://portal.digitalsafe.net/browse/SCMOD-5552): Autoscaler could stop scaling services  
+- **SCMOD-5552**: Autoscaler could stop scaling services  
         After encountering certain types of issues, such as networking downtime, the Autoscaler might have failed to recover even after the issue was resolved, and would no longer perform any scaling actions.
-
+- **SCMOD-7663**: Updated dropwizard and jetty versions
+        
 #### Known Issues
```
Comment (michael-bryson) [inline release-notes-1.6.0.md:12]: Add - None here

File: release-notes-1.6.0.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
 
 #### New Features
 
 #### Bug Fixes
 
-- [SCMOD-5552](https://portal.digitalsafe.net/browse/SCMOD-5552): Autoscaler could stop scaling services  
+- **SCMOD-5552**: Autoscaler could stop scaling services  
         After encountering certain types of issues, such as networking downtime, the Autoscaler might have failed to recover even after the issue was resolved, and would no longer perform any scaling actions.
-
+- **SCMOD-7663**: Updated dropwizard and jetty versions
```
Comment (michael-bryson) [inline release-notes-1.6.0.md]: Don't need to call this out

**Review by michael-bryson — CHANGES_REQUESTED** [review]
There is also a open pull request, as per the notes in https://portal.digitalsafe.net/browse/SCMOD-6966 this should be reviewed to see if it needs included in the release. If in doubt talk to the pull request author or Dermot.

File: release-notes-1.6.0.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 #### New Features
+ - None
 
 #### Bug Fixes
 
-- [SCMOD-5552](https://portal.digitalsafe.net/browse/SCMOD-5552): Autoscaler could stop scaling services  
+- **SCMOD-5552**: Autoscaler could stop scaling services  
         After encountering certain types of issues, such as networking downtime, the Autoscaler might have failed to recover even after the issue was resolved, and would no longer perform any scaling actions.
-
+
```
Comment (michael-bryson) [inline release-notes-1.6.0.md]: Whitespace?

**Comment by thilagasanthosh** [comment]
@michael-bryson ,
Verified with PR owners and that can be ignored for this release.

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**